### DISCUSSION TITLE
LS Emotion Home Base no Status update via remote Control

### DISCUSSION THREAD

**Discussion by Freasenx** [Koenkk/zigbee2mqtt#16640] [Devices]
I have a LS Emotion Home Base. [LS Emotion Home Base](https://zigbee.blakadder.com/L_S_7061288.html)
This has no direct support, so I built it with the instructions by myself.
I used the Light template for this.

```
const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
const exposes = require('zigbee-herdsman-converters/lib/exposes');
const reporting = require('zigbee-herdsman-converters/lib/reporting');
const extend = require('zigbee-herdsman-converters/lib/extend');
const ota = require('zigbee-herdsman-converters/lib/ota');
const tuya = require('zigbee-herdsman-converters/lib/tuya');
const utils = require('zigbee-herdsman-converters/lib/utils');
const globalStore = require('zigbee-herdsman-converters/lib/store');
const e = exposes.presets;
const ea = exposes.access;

const definition = {
    zigbeeModel: ['Emotion'],
    model: 'Emotion',
    vendor: 'LS Deutschland GmbH',
    description: 'L&S Emotion Home Base',
    // Note that fromZigbee, toZigbee and exposes are missing here since we use extend here.
    // Extend contains a default set of fromZigbee/toZigbee converters and expose for common device types.
    // The following extends are available:
    // - extend.switch
    // - extend.light_onoff_brightness
    // - extend.light_onoff_brightness_colortemp
    // - extend.light_onoff_brightness_color
    // - extend.light_onoff_brightness_colortemp_color
    extend: extend.light_onoff_brightness_colortemp(),
};

module.exports = definition;
```

The device works fine so far and can be controlled directly via Zigbee as well as HomeAssistant.

However, if I use the remote control to control the LEDs, the status, brightness or color temp. does not update.

Via the Refresh button on the Exposes tab in the Zigbee2MQTT Addon, I can update the single points and then it is correct again. 

As far as I understand, the device must report the changes itself. But it doesn't seem to do that.
Is there a possibility to update the values automatically ?

**Comment by mkaay** [comment]
I just ran into the same problem with the Home Base.
Not quite sure if this is the right way, but enabling reporting for the device seems to work.
Here is my config:
<img width="1622" alt="image" src="https://user-images.githubusercontent.com/121547/228933654-8b8c7313-2caa-42b4-8c23-634518b5fdb5.png">

**Comment by latinvm** [comment]
Have you found a way to select a different channel? My kitchen lights are spread over 3 channels, the home base only updates 1 of those.

**Reply by latinvm** [reply]
I have tried creating a group that contains all 3 endpoints, but this doesn't seem to work:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/7670602/1ab004fa-79f0-498d-8767-bc62e6389cb4)

Does anyone know a good tutorial on how to work with the DEV console?
![image](https://github.com/Koenkk/zigbee2mqtt/assets/7670602/ba17e7fd-01f1-48e6-b686-9dc6edf1f4f8)

**Comment by JungChristian** [comment]
> Have you found a way to select a different channel? My kitchen lights are spread over 3 channels, the home base only updates 1 of those.

Same problem here. 
The device seems to be officially supported right now ([see this issue](https://github.com/Koenkk/zigbee2mqtt.io/pull/2081)) but I also can only control 1 light / endpoint.
Endpoint 2 and 3 are reporting wrong states and also cannot be updated manually.
I also tried an external converter without any success.

**Comment by peppi911** [comment]
I had the same issues and tried everything to get it working.
Even reached out to L&S and they kept ignoring my emails.

However the fix was very simple.
After switching all my drivers to master mode, I was able to control each light group via the exposed endpoints that where there all the time 2 and 3 only start working after I switch the drivers to master mode.

There see https://www.youtube.com/watch?v=cYqvaUtX3j8&list=PLv-Mm0GV97Xw1KDCCtr5qwLCKcbERqYhd&index=7 how to do that.

**Reply by JungChristian** [reply]
Also worked for me. Thanks.

**Reply by AchimHome** [reply]
This also worked for me. recap:
- the pairing to the 4 channel remote was already there
- reset HomeBase unit and also delete the L&S entity within the homeassistant zigbee integration
- re-learn all the knobs of the 4 channel remote to the HomeBase unit
- connect to homeassistant with Zigbee (green mode) of the HomeBase
and as a last step:
- just switch all (Frankfurt, whatever) drivers to master mode (7 seconds on the knob, recognizable by: blinking blue forever) -> now light switch on/off commands etc. work from homeassistant
(if your distributor maintained you two switches paired as master/slave, you can leave this as is, HomeBase communicates with master and bridges to the slave unit).

You just have to ignore the red light on the HomeBase from now on. This seems to be normal (i suppose a minor protocol bug between Homebase and homeassistant).

**Reply by goar1** [reply]
do you use a l&S remote and home assistant in parallel to control your LEDs? I can confirm that my setup is almost same as yours. After pairing home base to HA the LED is green for 1-2 min then gets red again. The 4 channels appear in Zigbee2MQTT and also HA. The main issue is that the lights do not update in HA after they are switched with remote control. This makes the whole integration useless for automation etc.
